js/ElementRendererProvider.js

```diff
@@ -357,9 +358,17 @@ export default class ElementRendererProvider {
       if (config.style) {
         divEl.style.cssText = Utils.styleToCss(config.style);
       }
-
-      if (config.accessibility && config.accessibility.web) {
+  
+      if (accessibilityWeb) {
         Utils.appendAttributesFromObject(divEl, config.accessibility.web);
+      } else if ((accessibilityWeb && !config.accessibility.web.tabindex) || !config.accessibility) {
+        divEl.setAttribute("tabindex", "0");
+        divEl.addEventListener("keyup", function(event) {
+          event.preventDefault();
+          if (event.keyCode === 13 || event.keyCode === 32) {
+            window.open(`https://www.google.com/maps/search/?api=1&query=${config.la},${config.lo}`, '_blank');
```

Comment:
@FeelsGOOD I think here we should include the action click logic from lines below (i.e. `if (config.click && config.click.actions) { this.wrapAction(config.click); ...`